cost/aws/instance_cloudwatch_utilization/AWS Inefficient Instance Utilization using CloudWatch.pt

```diff
@@ -413,35 +419,70 @@ code <<-EOS
         "cpu_average": parseFloat(instance.cpu_average).toFixed(2),
         "mem_maximum": mem_maximum,
         "mem_minimum": mem_minimum,
-        "mem_average": mem_average
+        "mem_average": mem_average,
+		"instanceType": instance.instanceType
       }
     )
   }
 EOS
 end
 
+script "js_inefficient_utilization_metrics", type: "javascript" do
+  result "inefficient_utilization_results"
+  parameters "ds_aws_instance_size_map", "ds_merged_metrics", "param_avg_used_memory", "param_avg_cpu" 
+  code <<-EOS
+  var inefficient_utilization_results = []
+  for ( i = 0; i < ds_merged_metrics.length; i++){
+    var instance = ds_merged_metrics[i]
+	var cpu_average = instance.cpu_average
+	var mem_average = instance.mem_average
+	var next_instance_size = ds_aws_instance_size_map[instance.instanceType]["down"]
+	if(next_instance_size == null && typeof next_instance_size == "undefined" ){
+      next_instance_size="N/A";
+	}
+    if ((cpu_average < param_avg_cpu || mem_average < param_avg_used_memory)){
+      inefficient_utilization_results.push({
```

Comment:
please sort by region.

Reply:
Updated the code to sort the region list and tested successfully

Reply:
you should use the underscorejs sortby function